gatsby-config.js

```diff
@@ -117,6 +117,16 @@ module.exports = {
             },
           },
         },
+        tessen: {
+          product: 'DOC',
+          subproduct: 'TDOC',
+          segmentWriteKey: 'AEfP8c1VSuFxhMdk3jYFQrYQV9sHbUXx',
+          trackPageViews: true,
+          pageView: {
+            name: 'pageView',
+            category: 'DocPageView',
+          },
```

Comment:
I noticed we have a couple other properties on the docs site, namely `env` and `content_type`. Is that something we should also instrument using the [`getProperties`](https://github.com/newrelic/gatsby-theme-newrelic/tree/develop/packages/gatsby-theme-newrelic#tessen) option?

Reply:
I was unsure about this. We don't have any other environments and we've sorta done away with the "content type" concept. I'm happy to add additional metadata to these events, but there wasn't any additional acceptance criteria on the ticket 🤔

Reply:
I've added the environment and location to the event properties.